eldritch/test_encounters.py

```diff
@@ -459,6 +510,169 @@ def testLodge5Fail(self):
     self.assertEqual(self.char.clues, 0)
     self.assertEqual(self.char.place.name, "FrenchHill")
 
+  def testLodge6Join(self):
+    self.state.event_stack.append(encounters.Lodge6(self.char))
+    self.char.dollars = 3
+    choice = self.resolve_to_choice(MultipleChoice)
+    self.assertEqual(choice.choices, ["Yes", "No"])
+    choice.resolve(self.state, "Yes")
+    self.resolve_until_done()
+    self.assertEqual(self.char.dollars, 0)
+
+  def testLodge7Fail(self):
+    self.state.event_stack.append(encounters.Lodge7(self.char))
+    self.state.common.extend([items.Revolver38(), items.TommyGun()])
+    self.state.unique.extend([items.Cross(), items.HolyWater()])
+    with mock.patch.object(events.random, "randint", new=mock.MagicMock(return_value=3)):
+      self.resolve_until_done()
+    self.assertEqual(len(self.char.possessions), 0)
+    self.assertEqual(len(self.state.common), 2)
+    self.assertEqual(len(self.state.unique), 2)
+
+  def testLodge7PassCC(self):
+    self.state.event_stack.append(encounters.Lodge7(self.char))
+    self.state.common.extend([items.Revolver38(), items.TommyGun()])
+    self.state.unique.extend([items.Cross(), items.HolyWater()])
+    with mock.patch.object(events.random, "randint", new=mock.MagicMock(side_effect=[5, 3, 3])):
+      self.resolve_until_done()
+    self.assertEqual(len(self.char.possessions), 2)
+    self.assertEqual(len(self.state.common), 0)
+    self.assertEqual(len(self.state.unique), 2)
+
+  def testLodge7PassCU(self):
+    self.state.event_stack.append(encounters.Lodge7(self.char))
+    self.state.common.extend([items.Revolver38(), items.TommyGun()])
+    self.state.unique.extend([items.Cross(), items.HolyWater()])
+    with mock.patch.object(events.random, "randint", new=mock.MagicMock(side_effect=[5, 3, 5])):
+      self.resolve_until_done()
+    self.assertEqual(len(self.char.possessions), 2)
+    self.assertEqual(len(self.state.common), 1)
+    self.assertEqual(len(self.state.unique), 1)
+
+  def testLodge7PassUU(self):
+    self.state.event_stack.append(encounters.Lodge7(self.char))
+    self.state.common.extend([items.Revolver38(), items.TommyGun()])
+    self.state.unique.extend([items.Cross(), items.HolyWater()])
+    with mock.patch.object(events.random, "randint", new=mock.MagicMock(side_effect=[5, 5, 5])):
+      self.resolve_until_done()
+    self.assertEqual(len(self.char.possessions), 2)
+    self.assertEqual(len(self.state.common), 2)
+    self.assertEqual(len(self.state.unique), 0)
+
+  def testSanctum1Pass(self):
+    self.state.event_stack.append(encounters.Sanctum1(self.char))
+    self.char.lore_luck_slider = 0
+    self.state.unique.extend([items.Cross(), items.HolyWater(), items.TommyGun()])
+    with mock.patch.object(events.random, "randint", new=mock.MagicMock(return_value=5)):
+      choice = self.resolve_to_choice(CardChoice)
+    self.assertEqual(choice.choices, ["Cross", "Holy Water", "Tommy Gun"])
+    self.assertIn("Holy Water", choice.choices)
```

Comment:
How can this assertion ever fail if the previous assertion passes?

Reply:
What if a cosmic ray hits the memory between when the first and second asserts happen?